src/main/java/io/anserini/index/IndexInfo.java

```diff
@@ -25,7 +25,14 @@ public enum IndexInfo {
       "BM25",
       new String[] {
           "https://github.com/castorini/anserini-data/raw/master/CACM/lucene-index.cacm.20221005.252b5e.tar.gz" },
+<<<<<<< Updated upstream
       "cfe14d543c6a27f4d742fb2d0099b8e0"),
+=======
+      "cfe14d543c6a27f4d742fb2d0099b8e0",
+      IndexType.bm25,
```

Comment:
`IndexType.INVERTED` -> inverted index is what allows BM25, and tfidf, etc.

IIRC, all caps is the Java convention?